### PR TITLE
http.server: Check whether stream is closed before flushing

### PR DESCRIPTION
**Pull request by bakaiadam**

### CODE REVIEW

File: Lib/http/server.py [hunk 1]
```diff
@@ -422,7 +422,8 @@ def handle_one_request(self):
                 return
             method = getattr(self, mname)
             method()
-            self.wfile.flush() #actually send the response if not already done.
+            if not self.wfile.closed:
+                self.wfile.flush() #actually send the response if not already done.
```
Comment (vstinner) [inline Lib/http/server.py:426]:
```suggestion
                self.wfile.flush()  # Actually send the response, if not already done.
```

**Review by vstinner — COMMENTED** [review]
Hi, you should provide more information. What are you trying to do? What is the error message? How do you reproduce the issue?

**Comment by vstinner** [comment]
I suggest you to open an issue to describe the problem that you want to address.

**Comment by bakaiadam** [comment]
I got valueerrors for flushing on a closed file. I realized it can be solved by not closing the wfile in my code.

**Pull request closed without merging**